src/Couchbase.Lite.Shared/API/Sync/Replicator.cs

```diff
@@ -216,6 +217,14 @@ public void ResetCheckpoint()
             Config.Options.Reset = true;
         }
 
+        /// <summary>
+        /// Starts the replication
+        /// </summary>
+        public void Start()
+        {
+            Start(false);
+        }
+
         /// <summary>
         /// Starts the replication
         /// </summary>
```

Comment:
This doc comment doesn't say what the new option is for

Reply:
Updated the summary to reflect that..

Reply:
That's alright, but note the other doc comments which make use of the "param" key.  This is useful for doc generation.

Reply:
The param in line 231?

Reply:
Oh yes sorry I didn't notice it in the diff!